[bitnami/harbor] add ability to set serviceAccountName (#22686)
bitnami/charts: bitnami/harbor/README.md

| `nginx.containerSecurityContext.seccompProfile.type`      | Set container's Security Context seccomp profile                                                                         | `RuntimeDefault`        |
| `nginx.updateStrategy.type`                               | NGINX deployment strategy type - only really applicable for deployments with RWO PVs attached                            | `RollingUpdate`         |
| `nginx.lifecycleHooks`                                    | LifecycleHook for the NGINX container(s) to automate configuration before or after startup                               | `{}`                    |

> | `nginx.serviceAccountName`                                | Set the service account to be used by the pod, default if left empty                                                     | `""`                    |

Reviewer: ```suggestion
| `nginx.serviceAccountName`                                | Set the service account name for the NGINX pods                                                    | `""`                    |
```

| `portal.initContainers`                                    | Add additional init containers to the Harbor Portal pods                                                                 | `[]`                            |
| `portal.extraVolumeMounts`                                 | Optionally specify extra list of additional volumeMounts for the Harbor Portal pods                                      | `[]`                            |
| `portal.extraVolumes`                                      | Optionally specify extra list of additional volumes for the Harbor Portal pods                                           | `[]`                            |

> | `portal.serviceAccountName`                                | Set the service account to be used by the pod, default if left empty                                                     | `""`                            |

Reviewer: ```suggestion
| `portal.serviceAccountName`                                | Set the service account name for the Harbor Portal pods                                                    | `""`                            |
```

| `core.initContainers`                                    | Add additional init containers to the Harbor Core pods                                                                                                                                                                                                                                   | `[]`                          |
| `core.extraVolumeMounts`                                 | Optionally specify extra list of additional volumeMounts for the Harbor Core pods                                                                                                                                                                                                        | `[]`                          |
| `core.extraVolumes`                                      | Optionally specify extra list of additional volumes for the Harbor Core pods                                                                                                                                                                                                             | `[]`                          |

> | `core.serviceAccountName`                                | Set the service account to be used by the pod, default if left empty                                                                                                                                                                                                                     | `""`                          |

Reviewer: ```suggestion
| `core.serviceAccountName`                                | Set the service account name for the Harbor Core pods                                                                                                                                                                                                                     | `""`                          |
```

| `jobservice.initContainers`                                    | Add additional init containers to the Harbor Jobservice pods                                                                                                 | `[]`                                |
| `jobservice.extraVolumeMounts`                                 | Optionally specify extra list of additional volumeMounts for the Harbor Jobservice pods                                                                      | `[]`                                |
| `jobservice.extraVolumes`                                      | Optionally specify extra list of additional volumes for the Harbor Jobservice pods                                                                           | `[]`                                |

> | `jobservice.serviceAccountName`                                | Set the service account to be used by the pod, default if left empty                                                                                         | `""`                                |

Reviewer: ```suggestion
| `jobservice.serviceAccountName`                                | Set the service account name for the Harbor Jobservice pods                                                                                         | `""`                                |
```

| `registry.initContainers`                                               | Add additional init containers to the Harbor Registry pods                                                                                                                                                                                                                | `[]`                                                                                |
| `registry.extraVolumes`                                                 | Optionally specify extra list of additional volumes for the Harbor Registry pods                                                                                                                                                                                          | `[]`                                                                                |

> | `registry.serviceAccountName`                                           | Set the service account to be used by the pod, default if left empty                                                                                                                                                                                                      | `""`                                                                                |

Reviewer: ```suggestion
| `registry.serviceAccountName`                                           | Set the service account name for the Harbor Registry pods                                                                                                                                                                                                      | `""`                                                                                |
```

| `trivy.initContainers`                                    | Add additional init containers to the Trivy pods                                                                         | `[]`                                   |
| `trivy.extraVolumeMounts`                                 | Optionally specify extra list of additional volumeMounts for the Trivy pods                                              | `[]`                                   |
| `trivy.extraVolumes`                                      | Optionally specify extra list of additional volumes for the Trivy pods                                                   | `[]`                                   |

> | `trivy.serviceAccountName`                                | Set the service account to be used by the pod, default if left empty                                                     | `""`                                   |

Reviewer: ```suggestion
| `trivy.serviceAccountName`                                | Set the service account name for the Trivy pods                                                     | `""`                                   |
```
